lib/xml2Json.js

```diff
@@ -15,6 +27,8 @@ module.exports.process = function xml2Json(xmlString) {
         ]
     });
 
+    validateXml(xmlString, parser);
```

Comment:
Should we parse xml 2 times. I can suggest change exception hadling inside the Promise